bower.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "upup",
+  "homepage": "https://www.talater.com/upup/",
+  "authors": [
+    "Tal Ater <tal@talater.com> (https://www.talater.com/)"
+  ],
+  "description": "Control the content users see, even when they're offline",
+  "main": [
+    "dist/upup.min.js",
+    "dist/upup.sw.min.js"
+  ],
+  "moduleType": [
+    "globals"
+  ],
+  "keywords": [
+    "offline",
+    "offlinefirst",
+    "cache",
+    "serviceworker"
+  ],
+  "license": "MIT",
+  "devDependencies": {
+    "grunt": "^0.4.5",
+    "grunt-contrib-compress": "^0.13.0",
+    "grunt-contrib-connect": "^0.11.2",
+    "grunt-contrib-cssmin": "^0.13.0",
+    "grunt-contrib-imagemin": "^0.9.4",
+    "grunt-contrib-jshint": "^0.11.0",
+    "grunt-contrib-uglify": "^0.9.1",
+    "grunt-contrib-watch": "^0.6.1",
+    "grunt-markdox": "^1.2.1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/TalAter/UpUp.git"
+  }
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "Gruntfile.js",
+    "package.json",
+    "bower.json",
```

Comment:
bower.json is never ignored, see: http://bower.io/docs/creating-packages/#ignore